docs/DESIGN_TOKENS.md

```
@@ -0,0 +1,288 @@
+/* Feature-specific accent colors */
+:root {
+  --accent-conv: oklch(55% 0.2 285);      /* Conversion purple */
+  --accent-analytics: oklch(65% 0.15 230); /* Analytics blue */
+  --accent-speed: oklch(65% 0.2 145);      /* Speed green */
+}
+```
+
+#### `globals.css` - Base HTML Styling
+**Purpose**: Imports token files and applies base HTML styling
+
+**Contains:**
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)